examples/IrisClassification/predict.py

```diff
@@ -0,0 +1,68 @@
+import os
+import json
+import torch
+import ast
+from argparse import ArgumentParser
+
+from mlflow.deployments import get_deploy_client
+
+
+def convert_input_to_tensor(data):
+    data = json.loads(data).get("data")
+    input_tensor = torch.Tensor(ast.literal_eval(data[0]))
+    return input_tensor
+
+
+def predict(parser_args):
+    plugin = get_deploy_client(parser_args["target"])
+    input_file = parser_args["input_file_path"]
+    if not os.path.exists(input_file):
+        raise Exception("Unable to locate inuput file : {}".format(input_file))
+    else:
+        with open(input_file) as fp:
+            input_data = fp.read()
+
+    data = json.loads(input_data).get("data")
+    import pandas as pd
+
+    df = pd.read_json(data[0])
+    for column in df.columns:
+        df[column].astype("double")
```

Comment:
I think this line does nothing:

```python
import pandas as pd

df = pd.DataFrame({"a": [0]})
print("before:", df.dtypes)
df["a"].astype("double")
print("after:", df.dtypes)
```

outputs:

```
before:
 a    int64
dtype: object
after:
 a    int64
dtype: object
```

Reply:
added the assignment